components/managedidentity/locals.tf

```diff
@@ -1,11 +1,37 @@
 locals {
-  mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env
+  mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env == "jenkins_prod" ? "prod" : var.env == "jenkins_sbox" ? "sandbox" : var.env
```

Comment:
When is var env ever jenkins_ in this repo ? https://github.com/hmcts/version-reporter-services/blob/master/azure-pipelines.yaml#L95-L123

Reply:
sandbox is still jenkins_sbox

Reply:
I think you got things a little bit the wrong way round -- could you also put this in a different var so it doesn't overwriting existing functionality for prod/sandbox env